app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,65 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.services.TaskService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+
+    private final TaskService taskService;
+
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    @GetMapping
+    @ResponseStatus(HttpStatus.OK)
+    public List<Task> getTaskList() {
+        return taskService.getTaskList();
+    }
+
+    @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public ResponseEntity getTask(@PathVariable long id) {
+        Task task = taskService.getTask(id);
+        return ResponseEntity
+                .ok(task);
+    }
```

Comment:
HTTP 상태 코드를 `200 OK`로 전달하기 위해서 `ResponseEntity`로 하신 것 같습니다. 반환 타입이 `ResponseEntity`라서 무엇을 반환하는지 코드를 봐야지만 알 수 있네요.
HTTP 상태 코드는 `@ResponseStatus`로 이미 지정해주고 있으니 반환 타입은 `Task`로 쓰는 것은 어떨까요?
```suggestion
    @GetMapping("/{id}")
    @ResponseStatus(HttpStatus.OK)
    public Task getTask(@PathVariable long id) {
        return taskService.getTask(id);
    }
```